packages.yml

```diff
@@ -1,5 +0,0 @@
-packages:
```

Comment:
i think we may have a util or two in here, does this need to be deleted?

Reply:
oh that's a mistake from splitting them, no this should stay